datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -260,14 +260,20 @@ public SdxCluster getByCrn(String userCrn, String clusterCrn) {
         }
     }
 
-    public List<SdxCluster> getSdxClustersByCrn(String userCrn, String clusterCrn) {
+    public List<SdxCluster> getSdxClustersByCrn(String userCrn, String clusterCrn, boolean includeDeleted) {
```

Comment:
Is it possible that this method returns the same SdxCluster record twice in the list? I think that would lead to strange behaviour. Can you confirm that it doesn't return the same SdxCluster twice?

Reply:
It is not possible. As the CRN of these sdx clusters will be different.